shared/constants/metametrics.js

```diff
@@ -160,8 +160,12 @@
  * type is changed we identify the ledger_connection_type trait
  * @property {string} [NETWORKS_ADDED] - when user modifies networks we
  * identify the networks_added trait
+ * @property {string} [NFT_AUTODETECTION_ENABLED] - when Autodetect NFTs
```

Comment:
can you change the string value of the new entries to literals, aka 'nft_autodetection_enabled' which is being done in my new PR as well

Reply:
Only on the new entries, even though thats counter-intuitive, my PR will get the old ones.

Reply:
@brad-decker Good with me. Fixed here: [2588b23](https://github.com/MetaMask/metamask-extension/pull/14367/commits/2588b23a5673ee1000bd8664708c5ef4b508e723)